packages/List/src/list-item.scss

```diff
@@ -71,6 +82,86 @@
 
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-list-item-padding: var(--hop-space-inset-squish-md);
+  --ids-list-item-min-height: 3.5rem;
+  --ids-list-item-action-margin: var(--hop-space-inset-md);
+  --ids-list-item-border-radius: var(--hop-shape-rounded-md);
+
+  /* Compact */
+  --ids-list-item-padding-compact: var(--hop-space-inset-squish-md);
+  --ids-list-item-min-height-compact: var(--hop-space-inset-xl);
+
+  /* Member */
+  --ids-list-item-padding-member: var(--hop-space-inset-sm);
+  --ids-list-item-min-height-member: var(--hop-space-480);
+
+  /* Focused */
+  --ids-list-item-background-focused: var(--hop-neutral-surface-hover);
+  --ids-list-item-color-focused: var(--hop-neutral-text-hover);
+
+  /* Selected */
+  --ids-list-item-background-selected: none;
+  --ids-list-item-background-selected-multi: none;
+  --ids-list-item-font-weight-selected: var(--hop-body-sm-semibold-font-weight);
+  --ids-list-item-check-display: block;
+  --ids-list-item-check-margin: var(--hop-space-inset-sm);
+  --ids-list-item-check-color: var(--hop-neutral-icon);
+
+  /* Disabled */
+  --ids-list-item-color-disabled: var(--hop-neutral-text-disabled);
+  --ids-list-item-icon-color-disabled: var(--hop-neutral-icon-disabled);
+
+  /* Visual Identifier */
+  --ids-list-item-icon-size: var(--hop-space-inset-lg);
+  --ids-list-item-icon-margin: var(--hop-space-inline-sm);
+  --ids-list-item-svg-margin: var(--hop-space-inline-sm);
+  --ids-list-item-svg-color: var(--hop-neutral-icon);
+  --ids-list-item-icon-member-size: var(--hop-space-inset-xl);
+
+  /* Manager */
+  --ids-list-item-manager-color: var(--hop-neutral-icon-weak);
+  --ids-list-item-manager-margin: 0 0 0 var(--hop-space-inset-sm);
+
+  /* Checkbox */
+  --ids-list-item-checkbox-size: var(--hop-space-inset-md);
+  --ids-list-item-checkbox-radius: var(--hop-shape-rounded-sm);
+  --ids-list-item-checkbox-check: var(--hop-neutral-icon-active);
+  --ids-list-item-checkbox-margin: var(--hop-space-inline-sm);
+  --ids-list-item-checkbox-background: var(--hop-neutral-surface);
+  --ids-list-item-checkbox-background-disabled: var(--hop-neutral-surface-disabled);
+  --ids-list-item-checkbox-background-checked: var(--hop-neutral-surface-active);
+  --ids-list-item-checkbox-background-checked-disabled: var(--hop-neutral-surface-disabled);
+  --ids-list-item-checkbox-border: var(--hop-neutral-border);
+  --ids-list-item-checkbox-border-hover: var(--hop-neutral-border-hover);
+  --ids-list-item-checkbox-border-checked: var(--hop-neutral-border-active);
+  --ids-list-item-checkbox-check-width: var(--hop-space-inset-xs);
```

Comment:
The tokens should appear eventually once we have more components as we draw a sizing scale.

Reply:
Although in some situations you can manage to set an height using the font-size + padding.

Reply:
In the meantime I would use straight rem's this would highlight the need for these and prevent unintended side effects of using space tokens for sizing.